static/js/index.js

```diff
@@ -0,0 +1,147 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const setScrapingLogs = (logs) => {
+    const logsNode = document.getElementById('scraping-logs')
+    for (i in logs) {
+        pNode = document.createElement('p')
+        pNode.innerText = logs[i]
+        logsNode.appendChild(pNode)
```

Comment:
Create a temp div to append all the items you are creating. Attach it to the logsNode only in the end (should be more efficent)